### PR TITLE
SW-4565 / SW-3514 Make table headers title case and don't overflow

### PR DESCRIPTION
**Pull request by alexTerraformation**
Screenshot 2024-01-19 at 4 00 59 PM

### CODE REVIEW

**Comment by alexTerraformation** [comment]
publish rc

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terraware/web-components",
-  "version": "2.5.2",
+  "version": "2.5.3-rc.0",
```
Comment (karthikbtf) [inline package.json:3]: does this need to be undone or ok as is?

**Review by karthikbtf — APPROVED** [review]
🎉

File: package.json (hunk 1 above)
Comment (alexTerraformation) [reply package.json:3]: Yes, oops.  Thank you!

Comment (alexTerraformation) [reply package.json:3]: @nickgraz I've not used this before.  Will this get replaced with the new version upon release in NPM?

Comment (nickgraz) [reply package.json:3]: OK as is, the next version will be `2.5.3`, which is tagged when this is merged in. `2.5.3-rc.0` is the first release candidate for `2.5.3` and the version here will get overwritten when the branch is merged.

**Review by nickgraz — APPROVED** [review]

**Pull request merged**